src/CommunityCommons/javasource/communitycommons/Misc.java

```diff
@@ -602,41 +602,36 @@ public static Language getDefaultLanguage(IContext context) throws CoreException
 	public static boolean mergePDF(IContext context,List<FileDocument> documents,  IMendixObject mergedDocument ) throws IOException{
 		if (Constants.getMergeMultiplePdfs_MaxAtOnce() <= 0 || documents.size() <= Constants.getMergeMultiplePdfs_MaxAtOnce()) { 
 		
-			int i = 0;
-			PDFMergerUtility  mergePdf = new  PDFMergerUtility();
-			for(i=0; i < documents.size(); i++)
-			{
-			    FileDocument file = documents.get(i);
-			    try (
```

Comment:
Agree that this inner try-with was nested a little deep but removing it causes the issue described here: https://github.com/mendix/CommunityCommons/issues/65

Reply:
Thanks for reporting this!